change/@fluentui-global-context-196e945e-ac56-46f6-957b-b5486e44b586.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "chore: migrate to monosize 0.4",
```

Should all of these read 0.5?

```suggestion
  "comment": "chore: migrate to monosize 0.5",
```